liblangutil/SourceReferenceExtractor.cpp

```diff
@@ -30,7 +30,8 @@ using namespace solidity::langutil;
 SourceReferenceExtractor::Message SourceReferenceExtractor::extract(
 	CharStreamProvider const& _charStreamProvider,
 	util::Exception const& _exception,
-	std::variant<Error::Type, Error::Severity> _typeOrSeverity
+	std::variant<Error::Type,
```

Comment:
Because I haven't see us using this that often for `std::variant` and the length seems short enough to not benefit readability. But I'm fine either way, was just wondering why haha

Reply:
Ah, no, you're right! My mistake. Not sure how I did not notice this is in the middle of a variant :) I thought I was a separate arg, but looking at it now, it's obviously not.